### PR TITLE
Fix file scan ignoring DVD image files (iso, img, m2ts)

### PR DESCRIPTION
**Pull request by dkrahmer**
#### Database Migration
NO

#### Description
Fix file scan ignoring DVD image files (iso, img, vob, m2ts)
Always allow DVD and BR file types without analysis, instead of detecting as 0 runtime.
Bonus bug fix: Add support for video files with non-lowercase extensions.

#### Issues Fixed or Closed by this PR

* #809

### CODE REVIEW

**Comment by Qstick** [comment]
Couple comments, also rebase this on latest to get rid of the test failures. Thanks for the contribution!

File: src/NzbDrone.Core/MediaFiles/MovieImport/DetectSample.cs [hunk 1]
```diff
@@ -32,16 +34,25 @@ public DetectSampleResult IsSample(Movie movie, string path, bool isSpecial)
...
+                    return DetectSampleResult.NotSample;
+                }
+
+                if (extension.Equals(".strm", StringComparison.InvariantCultureIgnoreCase))
+                {
+                    _logger.Debug("Skipping sample check for .strm file");
+                    return DetectSampleResult.NotSample;
+                }
+
+                if (new Quality[] { Quality.DVD, Quality.Bluray720p }.Contains(MediaFileExtensions.GetQualityForExtension(extension)))
```
[the remark this answers is not in the document]
Comment (dkrahmer) [reply src/NzbDrone.Core/MediaFiles/MovieImport/DetectSample.cs]: Done.

[the remark this answers is not in the document]
Comment (dkrahmer) [reply src/NzbDrone.Core/MediaFiles/MovieImport/DetectSample.cs]: Done. Added 3 unit tests.

**Comment by Qstick** [comment]
Thanks, seems your rebase went a bit screwy at some point as its bringing in aphrodite commits, try this

`git rebase upstream/aphrodite -i`

and then 

`git push -f` - make sure to force push to re-write the history on the branch

**Comment by dkrahmer** [comment]
Everything should be in order now.

**Pull request merged**